feeds/wifi-ax/hostapd/files/wps-hotplug.sh

```diff
@@ -1,5 +1,33 @@
 #!/bin/sh
 
+wps_flash_leds() {
+	local ubusobjs=$1
+	local finished=0
+
+	STATE=$(/sbin/leds.sh led_get_state)
+
+	/sbin/leds.sh led_wps
+
+	. /usr/share/libubox/jshn.sh
+	while [ $finished -eq 0 ]; do
+		for ubusobj in $ubusobjs; do
+			json_init
+			json_load "$(ubus -S call $ubusobj wps_status)"
+			json_get_vars pbc_status
+			[ "$pbc_status" = "Timed-out" -o "$pbc_status" = "Disabled" ] && {
+				finished=1
+			}
+		done
+		sleep 1
+	done
+	if [ "$STATE" = "led_actions_off" ] ; then
+                /sbin/leds.sh led_off
+        else
+                /sbin/leds.sh $STATE
```

Comment:
this should still be calling led_wlan like the r14 version does

Reply:
modified the script as per R14